### PR TITLE
renamed LEPStatus/Limited English Proficiency to EnglishLearner/Engli…

### PR DESCRIPTION
**Pull request by dsnyder-fairway**
…sh Learner

### CODE REVIEW

File: common/src/main/java/org/opentestsystem/rdw/reporting/common/model/DimensionType.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
     Overall(-1, "Overall"),
     Gender(1, "Sex"),
     Ethnicity(2, "Ethnicity"),
-    LEP(3, "LEPStatus"),
+    LEP(3, "EnglishLearner"),
```
Comment (dsnyder-fairway) [inline common/src/main/java/org/opentestsystem/rdw/reporting/common/model/DimensionType.java]: @jtreuting any reason why we cannot change this text? Comments above state 
```
* Note: The codes are associated with SBAC data terms defined by Organization Exam Export
* functionality.
```

Comment (jtreuting) [reply common/src/main/java/org/opentestsystem/rdw/reporting/common/model/DimensionType.java]: @malaffoon do you remember if this was to align with CEDS?

Comment (malaffoon) [reply common/src/main/java/org/opentestsystem/rdw/reporting/common/model/DimensionType.java]: Yes, CEDS refers to that as LEP, pretty definitively. Bit surprised they'd want to change that, but i guess if CDE wants it to be something else, they can do whatever they want.

**Comment by dsnyder-fairway** [comment]
Closing for now, I'll open another PR since which will be the reverse of this (changing English Learner to LEP / Limited English Proficiency)

**Pull request closed without merging**